parquet-column/src/main/java/org/apache/parquet/column/statistics/BinaryStatistics.java

```diff
@@ -46,8 +46,8 @@ public void mergeStatisticsMinMax(Statistics stats) {
 
   @Override
   public void setMinMaxFromBytes(byte[] minBytes, byte[] maxBytes) {
-    max = Binary.fromByteArray(maxBytes);
-    min = Binary.fromByteArray(minBytes);
+    max = Binary.fromReusedByteArray(maxBytes);
+    min = Binary.fromReusedByteArray(minBytes);
```

Comment:
We don't have control over where the byte arrays come from, so the safe thing is to use the reused form. If we changed the API and could make guarantees about it, then I think we could do otherwise.